### ISSUE TITLE
caskroom.io seems to have been hacked (again)

### ISSUE THREAD

**Issue by itaisteinherz** [Homebrew/homebrew-cask#62508]
#### Description of issue

It seems like https://caskroom.io has been hacked (again - #53375, #52164, [etc.](https://github.com/Homebrew/homebrew-cask/issues?utf8=✓&q=is%3Aissue+is%3Aclosed+caskroom.io+hacked)), and currently prompts visitors to buy Instagram followers 🤦🏻‍♂️

<img width="720" alt="caskroom hacked" src="https://user-images.githubusercontent.com/22768990/56852201-b7800080-6920-11e9-8709-df52d20da266.png">

I also [tweeted about this](https://twitter.com/itaisteinherz/status/1122171374502600704).

**Comment by ran-dall** [comment]
@itaistenherz https://github.com/Homebrew/homebrew-cask/issues/51928#issuecomment-421189147

While I (and the rest of the HBC maintainers and committers, I'm sure) appreciate your enthusiasm; maybe next time it would be wise to check through all closed and related issues before trying to start a commotion. 😁

**Comment by itaisteinherz** [comment]
@ran-dall Thanks for the heads up, and sorry for causing unnecessary drama 😬

**Comment by ran-dall** [comment]
@itaisteinherz No problem!🤙 Don't worry, it happens!

**Comment by vitorgalvao** [comment]
I’ll add there is no “again”. Not only was the site never hacked in the first place, we never got it back once it went to those spammers.

That’s an easy mistake; it’s understandable. What I’m failing to understand is how you found those other issues (even before the edits, you had already posted them) but didn’t get the answer. Did you just link to them without reading them?

I’m glad to see they’ve changed the site. At least now it’s clear it’s not related to us.

**Comment by itaisteinherz** [comment]
@vitorgalvao I went over some of the issues, which were closed by linking to other issues. I didn't go on to reading the rest of the issues unfortunately, which lead me to believe that this "hack" might be a real issue and should be reported for further investigation.
Also, I've never seen this _Instagram followers_ thing up until now, despite the fact that I've been using Homebrew and Casks for nearly 3 years, which also turned on some red lights for me.
